### DISCUSSION TITLE
Connecting an Amiga external floppy drive to greaseweazle possible?

### DISCUSSION THREAD

**Discussion by melomonday** [keirf/greaseweazle#409] [Q&A] [answered]
I have a common Amiga external floppy drive (supradrive) with a 23 pin connector cable (male) and a 23 pin socket/connector (female) at the back of the drive. Is it possible to connect this drive to greaseweazle 4.1, using some kind of adapter from 23 pin to 34 pin (e.g. the Retroninja external floppy adapter)?

**Comment by keirf** [comment]
I would expect it to work, yes.

**Comment by melomonday** [comment]
Unfortunately, I couldn't get it to work with the set-up in the picture. The "gw read --format=amiga.amigados MyAmigaDisk.adf --drive=1" command returned "Command Failed: GetFluxStatus: No Index" (I tried drive 0 and 2 as well) but no response and the drive seemed dead. I tried connecting the regular connector cable of the drive to a powered-up Amiga at the same time to make sure the drive had power. This made the drive come alive but still no luck with the read command. Any ideas?
![IMG_6554](https://github.com/keirf/greaseweazle/assets/159567820/357a3e94-55fd-4bd0-8038-6be2e5244fef)

**Comment by keirf** [comment]
Oh I see yes that's the daisy chain connector. Actually now I think about it this probably won't work. But you could extract the drive mechanism from within it's enclosure and you'd probably find a regular 34 pin connector in there.

**Comment by melomonday** [comment]
Good idea, thanks. I did find a regular 34 pin connector and power connector that I could connect greaseweazle to. However still the same error message when running the read command. It seems the drive doesn't get power. I'm wondering if the problem could be that the power cable that was used inside the disk enclosure connects to that small circuit board with an on-off-protect switch. If I just connect it straight like in the picture perhaps the drive doesn't understand that it's "on". 
![IMG_6555](https://github.com/keirf/greaseweazle/assets/159567820/142ecc31-429f-4b04-9329-307ae8286701)

**Reply by keirf** [reply, marked as answer]
That drive might need 12v in which case unfortunately you can't power it from the Greaseweazle. Instead you could power it via the Amiga and that adapter board?

**Comment by melomonday** [comment]
It works, thanks a lot! Becomes a bit unpractical to have to power up the amiga every time but at least I know that the drive works with greaseweazle.
